symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -35,6 +36,27 @@ public function registerBundles()
         }
     }
 
+    public static function bootstrapEnvironment(string $env = null)
+    {
+        $env = $env ?? $_SERVER['APP_ENV'] ?? null;
+        $projectDir = __DIR__.'/..';
```

Comment:
should be `$this->getProjectDir()` instead.

Reply:
@fabpot the function is static. Creating the AppKernel happens at later point after the env has been bootstrapped.